packages/oscal-react-library/src/components/OSCALControlProse.js

```diff
@@ -309,10 +308,10 @@ export function OSCALReplacedProseWithParameterLabel(props) {
 
   if (!props.isImplemented) {
     return (
-      <NotImplementedStatement>
+      <NotSpecifiedTypography>
```

Comment:
How does changing from a `Stack` to a `Typography` preserve the correctness of this? Are we sure this doesn't reintroduce the errors? And can you explain to me how/why?

Reply:
You're right. I was worried about that. Ideally I'd set the `Typography` component to something like a `<div>`, but that didn't seem to work.